### PR TITLE
added documentation for ContactPageModel and all fields

### PR DESCRIPTION
**Pull request by crex424**
I added documentation for all the fields. Let me know if any are inaccurate, or poorly written.

### CODE REVIEW

File: PC2/Models/ContactPageModel.cs [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+        /// <summary>
+        /// The contact's phone number
+        /// </summary>
         public string Phone { get; set; }
+        /// <summary>
+        /// The subject of the message
+        /// </summary>
         public string Subject { get; set; }
+        /// <summary>
+        /// The message for the contact
```
Comment (JoeProgrammer88) [inline PC2/Models/ContactPageModel.cs:25]: Would "The message from the contact" be a little more clear?

**Review by JoeProgrammer88 — CHANGES_REQUESTED** [review]

File: PC2/Models/ContactPageModel.cs [hunk 2]
```diff
@@ -1,11 +1,29 @@
 ﻿namespace PC2.Models
 {
+    /// <summary>
+    /// Represents a contacts information
```
[the remark this answers is not in the document]
Comment (crex424) [reply PC2/Models/ContactPageModel.cs:4]: We definitely can change the documentation. I will get right on enacting the changes you have proposed.

File: PC2/Models/ContactPageModel.cs (hunk 1 above)
Comment (crex424) [reply PC2/Models/ContactPageModel.cs:25]: I could definitely make that description clearer. Will refactor soon.

File: PC2/Models/ContactPageModel.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (crex424) [reply PC2/Models/ContactPageModel.cs]: Alrighty, I have refactored the documentation cited above. Let me know if you want any further changes.

**Review by JoeProgrammer88 — APPROVED** [review]
I think with the revised documentation we are good to merge these changes

**Comment by crex424** [comment]
Alrighty, added some more details to the documentation for the Message field. Let me know if the added details are accurate.

**Pull request merged**